arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist

```diff
@@ -493,6 +493,14 @@
 				<key>descriptionText</key>
 				<string>Display a map with an OpenStreetMap basemap.</string>
 			</dict>
+			<dict>
+				<key>displayName</key>
+				<string>Display WFS</string>
```

Comment:
Display name is little different than what we have in issue and read me doc. We can either update sample name or readme doc to maintain uniformity.

Reply:
I followed the naming patterns found for WMS Layer. Would you prefer the name to be `Display a WFS layer`? @saratchandrakarumuri @philium

Reply:
I would go with "Display a WFS layer" as that is the name of the issue. Our standard practice is to title the sample implementation issue with the name of the sample.